Sources/LeafKit/LeafParser/LeafParser.swift

```diff
@@ -1,124 +1,33 @@
-extension String: Error {}
-
-private struct TagDeclaration {
```

Comment:
* Annotations/Rearrangment
* Explicitly makes `LeafParser` internal
* Consolidates and moves helper structs internal to `LeafParser`